### ISSUE TITLE
Bonferroni-corrected post-hoc comparisons for interactions in classical RM ANOVA

### ISSUE THREAD

**Issue by IndrajeetPatil** [jasp-stats/jasp-desktop#423]
It will be nice to have this feature in JASP. Thanks.

**Comment by jonathon-love** [comment]
Yes! Absolutely. Coming soon!

**Comment by jonathon-love** [comment]
oh, wait, in interactions? not sure about that.

**Comment by IndrajeetPatil** [comment]
Ah, okay. SPSS does not have it as a default and MedCalc has it but, again, you need to pay for it. So it would be lovely if this would be a part of JASP as it's really common to conduct post-hoc comparisons for something like a 2-by-2 ANOVA.
In any case, thanks a lot for your reply and also for all the great work JASP team is doing.

**Comment by IndrajeetPatil** [comment]
Will this finally be implemented?

I would really appreciate it if this can be done. Currently, I can't think
of any other stat software package that can do this. And it is so often
that researchers follow up on an interaction effect with post-hoc
comparisons in RM ANOVA designs. So it will be super-useful to many people.


Thanks.

**Comment by jonathon-love** [comment]
let's ask @raviselker about this one.

i think he's on holidays for the next couple of weeks, so you may not get a response for a while.

as you can imagine, the ratio of our resources to the number of features that would be good to implement isn't that high. hopefully as JASP grows, and with it our community and funding, we will be able to pursue these good ideas more expediently.

**Comment by IndrajeetPatil** [comment]
Hi, 

Just a follow-up on this question: I see that this thread has been closed, but I still don't see any option for carrying out post-hoc tests for *interactions,* which is what I was querying about. The current option only does this for *main effects.*

Is there any plan to include this in future version of JASP?

Thanks,
Indrajeet

**Comment by raviselker** [comment]
This issue is still open. I closed a different issue that was a duplicate of this one. We still want to do this at some point, but as it will not be trivial to implement this we have other priorities at the moment.

**Comment by IndrajeetPatil** [comment]
Okay, good to know that this will be implemented in future. Thanks.

**Comment by IndrajeetPatil** [comment]
Hi,
Sorry for being so pesky, but I think this will really be a valuable addition to JASP.
So I was wondering if this was going to be a part of the upcoming JASP 0.8.0.0 release.

**Comment by renenob** [comment]
It can be done in spss via changing the syntax (minor). You may check how2stats youtube video. It will be awesome if JASP can implement this. Post moderation slopes analysis too, in the case of multiple regression.

**Comment by IndrajeetPatil** [comment]
Yes, I know it can be done with SPSS, but what point is having JASP if you have to go to SPSS for some of the things?! 
It'd be really great if JASP can do this.

**Comment by R180** [comment]
Did this ever get addressed in JASP? It does seem to exist in R, and in some other open-access R-based stats programs.

**Comment by JohnnyDoorn** [comment]
Hi Richard,
It's definitely still on the list! There have been some other, more pressing, features of ANOVA we wanted to implement first, but as those are (almost) done, it shouldn't be a long wait!
Cheers
Johnny

**Comment by EJWagenmakers** [comment]
@JohnnyDoorn: maybe good to turn to this soon; some people are eagerly awaiting this feature.
